src/vs/workbench/browser/media/style.css

```diff
@@ -51,6 +51,23 @@ body.web {
 	position: relative;
 	z-index: 1;
 	overflow: hidden;
+	height: 100vh;
```

Comment:
@eamodio this change is not clear to me and I am not sure about the consequences. Is it really needed and why this format and not `100%`?

Reply:
By using `vh/wh` we can avoid having to manually size the container and avoids other issues with using % where all the parents need to be 100%, etc. So 100vh/wh will always size the container to the viewport height/width.

Reply:
@eamodio still, we used to set `position: relative` programmatically, no?

Reply:
We did, but it is set on `monaco-workbench` -- is there a need for the inline-style?

Reply:
True, ignore.